spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/ContainerRegistryController.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.dataflow.server.controller;
+
+import java.util.List;
+import java.util.Map;
+
+import org.springframework.cloud.dataflow.container.registry.ContainerRegistryService;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/container-registry")
+public class ContainerRegistryController {
+
+	private ContainerRegistryService containerRegistryService;
+
+	public ContainerRegistryController(ContainerRegistryService containerRegistryService) {
+		this.containerRegistryService = containerRegistryService;
+	}
+
+	@RequestMapping(value = "/config", method = RequestMethod.GET)
```

Comment:
should not use `config` on this level as we're mapping it to `registryName` later.

Reply:
Removed this controller as not needed anymore.